src/main/java/seedu/address/model/charge/Charge.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.model.charge;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the daily amount chargeable of a Pet in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Charge {
+    public static final String MESSAGE_INVALID_CHARGE_FORMAT = "Charge should be formatted as [number] or "
+             + "[number1.number2] where number2 can be up to 2 decimal places!";
+
+    /**
+     * Regex to match charge format.
+     * eg: "80.40".
+     */
+    public static final String VALIDATION_REGEX =
+            "\\d+[.]?\\d{0,2}";
+
+    private final Double perDayCharge;
+
+    /**
+     * Constructs an {@code Charge}.
+     *
+     * @param charge A valid per day charge.
+     */
+    public Charge(String charge) {
+        requireNonNull(charge);
+
+        this.perDayCharge = Double.parseDouble(charge);
+    }
+
+    /**
+     * Returns the value of a valid charge per day
+     * @return perDayValue as an integer.
```

Comment:
Should the documentation here reflect Double instead?

Reply:
Right! Thanks for pointing it out!